packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2609,32 +2609,40 @@ export class ContainerRuntime
 		await this.pendingStateManager.applyStashedOpsAt(message.sequenceNumber);
 	}
 
+	/**
+	 * Processes the op.
+	 * @param messageArg - Sequenced message for a distributed document.
+	 * @param local - true if the message was originally generated by the client receiving it.
+	 */
 	public process(messageArg: ISequencedDocumentMessage, local: boolean) {
```

Comment:
snit: "Arg" in `messageArg` is redundant.

Reply:
It may be intentionally named like that, look at how many variables named like "message" there are. This is the one that was passed in.  That said, I don't see anyone using `message` yet so could be renamed if so desired.

Reply:
@pragya91 consider doing this for clarity/safety:
```suggestion
	public process({ ...messageCopy }: ISequencedDocumentMessage, local: boolean) {
```